### PR TITLE
DOCS-1418-SIEM-logs

### PR DESCRIPTION
**Pull request by bmckercher123**
Document logs that can be exported by SIEM.

Product Version(s):
CC - 3.16, 3.17, 3.18
CE - 3.16, 3.17, 3.18

Issue:
https://tigera.atlassian.net/browse/DOCS-1418

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise/visibility/elastic/archive-storage.mdx [hunk 1]
```diff
@@ -22,6 +22,14 @@ This how-to guide uses the following {{prodname}} features:
 
 - **LogCollector** resource
 
+## Before you begin
+
+**Supported logs for export**
+
+- Syslog - flow, dns, idsevents, audit
+- Amazon S3 - l7, flow, dns, runtime, compliance, audit
```
Comment (penkeysuresh) [inline calico-enterprise/visibility/elastic/archive-storage.mdx]: [In the ticket](https://tigera.atlassian.net/browse/DOCS-1418?focusedCommentId=64261) it's mentioned that we will not be adding runtime for enterprise, did that change ?

File: calico-cloud/visibility/elastic/archive-storage.mdx [hunk 2]
```diff
@@ -22,6 +22,14 @@ This how-to guide uses the following {{prodname}} features:
 
 - **LogCollector** resource
 
+## Before you begin
+
+**Supported logs for export**
+
+- Syslog - flow, dns, idsevents, audit
+- Amazon S3 - l7, flow, dns, runtime, compliance, audit
```
Comment (penkeysuresh) [inline calico-cloud/visibility/elastic/archive-storage.mdx]: [In the ticket](https://tigera.atlassian.net/browse/DOCS-1418?focusedCommentId=64261) it's mentioned that we will not be adding compliance to this document.

**Review by penkeysuresh — CHANGES_REQUESTED** [review]

File: calico-enterprise/visibility/elastic/archive-storage.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise/visibility/elastic/archive-storage.mdx]: Sorry, I forgot to commit my changes...you should see "runtime" only the CC version of the docs.

**Review by penkeysuresh — APPROVED** [review]
LGTM

**Review by girishshankaran — APPROVED** [review]
LGTM

File: calico-cloud/visibility/elastic/archive-storage.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-cloud/visibility/elastic/archive-storage.mdx]: As previous comment.

**Pull request merged**